web/www/horas/Latin/Sancti/12-27.txt

```diff
@@ -225,23 +233,23 @@ $Deo gratias
 Exiit * sermo inter fratres, quod discipulus ille non moritur; et non dixit Jesus, Non moritur: sed, Sic eum volo manere, donec veniam.
 
 [Octava 2]
-!Commemoratio octavae S. Joannis.
-Ant. Iste est Joannes, qui supra pectus Domini in cena recubuit: beatus Apostolus, cui revelata sunt secreta caelestia.
+!Commemoratio Octavae S. Joannis
+Ant. Iste est Joannes, qui supra pectus Domini in caena recubuit: beatus Apostolus, cui revelata sunt secreta coelestia.
```

Comment:
The only point can be, that this Ocatvae was abandoned, when writing (in 1902) as 'coena' and 'coelestia'.
Of course this provides some inconsistency, which can be? cured in code.